news/_posts/2020-03-12-ps17-minor-release-lifecycle.md

```diff
@@ -0,0 +1,125 @@
+
+# The journey begins: Kanban and scope
+
+Each minor version is defined by a feature scope, and this scope is defined as a number of github issues that we put into a [Kanban](https://help.github.com/en/github/managing-your-work-on-github/about-project-boards) (see [1.7.7 Kanban](https://github.com/PrestaShop/PrestaShop/projects/7)). How these issues are picked or sorted is the responsibility of the Product Team which spends a lot of time gathering feedback about all PrestaShop users to make sure the next minor version addresses the most important matters and needs.
+
+For example 1.7.7.0 scope contained, but not only:
+- Compatibility with php 7.3
+- Migration and rework of Back-office Order pages
+- Advanced currencies management
+
+Although it is very hard to estimate the size of this scope, we try to size it in order for the development phase to last 4 months.
+
+When this scope is completed - i.e all github issues have reached the "Done" column of the Kanban - we trigger the [Feature Freeze](https://en.wikipedia.org/wiki/Freeze_(software_engineering)).
+
+We however accept that some issues remain in the Kanban, outside of "Done" column, only if they are bugs, no features. This is a Feature Freeze, not a Bug Freeze.
+
+# Feature Freeze
+
+Feature freeze means that we freeze the development of the version and enter a phase of stabilization, whose aim is naturally the release some time later.
+
+We create a git branch from `develop` branch which will carry the work to be done until the release (for 1.7.7.0, the branch name is `1.7.7.x`). From this moment, only bug fixes can be merged into this branch.
+
+Maintainers create an Alpha Build from the `1.7.7.x` branch codebase and deliver it to QA team which starts a huge test campaign. The goal of this campaign is to find and register all regressions[1] of this build.
+
+When the test campaign ends, we populate the Kanban with all found bugs.
+We fix all important regressions as fast as possible because when this point is reached, we can launch the [Beta phase](https://en.wikipedia.org/wiki/Software_release_life_cycle#Beta). Although it depends heavily on the number of found regressions, we try to size this phase to last one month.
+
+For PrestaShop 1.7.7.0, QA team is currently finishing the test campaign.
+
+# Beta period
+
+When the branch reaches a point of maturity, which means only minor or trivial issues remain to be fixed, developers team create a Beta Build using the `1.7.7.x` branch codebase.
+
+This Beta build is provided to anyone and we open a one-month long Beta period where we ask **your** help.
+
+During this one month, we continue testing and fixing the 1.7.7.x (following the stabilization goal) but we know that we can only test and imagine a limited amount of usecases. The community however knows better than us all the possible ways to use PrestaShop to build a business.
+
+So **you can help us** making this release a better and more stable release by testing this Beta build.
```

Comment:
@ttoine @eternoendless Feel free to improve this "this is how you can beta test" part